topbeat/docs/fields.asciidoc

```diff
@@ -168,12 +167,12 @@ type: int
 
 The amount of CPU time spent in involuntary wait by the virtual CPU while the hypervisor was servicing another processor. Available only on Unix.
 
-[float]
+
 === cpus Fields
 
 This group contains CPU usage per core statistics.
 
-[float]
```

Comment:
@dedemorton Thanks for the elaboration. I did an intermediate "hack" by introdcuing a "prefix" flag which can be set in the fields.yml file to generate the [float] tag. The reason I didn't fully automate it because currently it seem the usage of [float] tags is not to 100% logical. I assume we have 1-2 too many of the tags inside which leads to this (see my comment in the code).

If we could find a shorter version as you described above, I'm happy to find a solution to automate this (should be possible). With metricbeat this is getting even more important.